consensus/src/marshal/actor.rs

```diff
@@ -772,7 +791,7 @@ impl<B: Block, E: Rng + Spawner + Metrics + Clock + GClock + Storage, V: Variant
         // Check finalized blocks.
         match self.finalized_blocks.get(ArchiveID::Key(&commitment)).await {
```

Comment:
If the panic does not include any extra information except for the error, then doing `self.finalized_block.get(...).await.expect("<reason this shouldn't fail>");`

Right now the `{e}` doesn't give you the error source chain, if there is any.